providers.tf

```diff
@@ -1,5 +1,5 @@
 provider "aws" {
-  region = var.region
+  region = "us-east-1"
```

Comment:
it was `default = "us-east-2"`  are you sure you want to change it ?

Reply:
It was US East 2 when we used to manage the EKS cluster. Now it's gone, all the resources left on the AWS CloudBees account are in `us-east-1` hence the change.